### ISSUE TITLE
Bug: Registry TreeViewItem TextBlocks are empty 

### ISSUE THREAD

**Issue by Mustafa5040** [0x5bfa/RegistryEditor#2]
![image](https://user-images.githubusercontent.com/52122907/235425142-abf7d5a5-67a2-4767-a8e7-67431cf5cdfa.png)
As you can see in the screenshot, registry location/directory names are empty in the left pane.
I built the app according to the instructions; x64 DEBUG Windows 11.

**Comment by 0x5bfa** [comment]
Oh my god, it's broken. That issue still happen even though you rebuild or clean-build?

**Comment by Mustafa5040** [comment]
Yeah, i tried cleaning and rebuilding but it didn't work. It's still broken unfortunately.

**Comment by Mustafa5040** [comment]
> Oh my god, it's broken. That issue still happen even though you rebuild or clean-build?

Yeah, i tried cleaning and rebuilding but it didn't work. It's still broken unfortunately.

**Comment by 0x5bfa** [comment]
I cannot repro this issue.

1. Clean install windows 11 pro insider preview canary
2. Clean install Visual Studio 2022
3. Re-clone RegistryValley
4. Build
(Clean installs weren’t for this though)

![image](https://user-images.githubusercontent.com/62196528/236674758-31862688-2898-4dbb-856a-334278b1c140.png)

**Comment by 0x5bfa** [comment]
This is probably an issue from WinAppSdk.
I'm still caring this issue but closing.

Thank you for reporting!

**Comment by 0x5bfa** [comment]
Are you still able to reproduce this issue?

**Comment by Mustafa5040** [comment]
> Are you still able to reproduce this issue?

Hi, it's been 3 years since i last tried. But i'll try again as soon as possible.

**Comment by 0x5bfa** [comment]
Haha sorry for the inconvenience, and thanks!
